afl-fuzz.c

```diff
@@ -6513,7 +6677,9 @@ static u8 fuzz_one(char** argv) {
 #ifdef AFL_USE_MUTATION_TOOL
       int mutated = 0;
       if (UR(64) < P_MUTATION_TOOL) {
-        mutated = use_mutation_tool(&out_buf, &temp_len);
+        // mutated = use_mutation_tool(&out_buf, &temp_len);
+        /* Hit the mutation server. */
+        mutated = post("localhost", 4448, &out_buf, &temp_len);
```

Comment:
Open question: how should the logic interact with the current `use_mutation_tool` function? We can an operation that picks either that function or the server with some probability, or just have another flag to configure toggling.